client/src/util/auth.jsx

```diff
@@ -280,7 +277,7 @@ export const requireAuth = (Component) => {
 };
 
 // Throw error from auth response, so it can be caught and displayed by UI
-function handleError(response) {
-  if (response.error) throw response.error;
-  return response;
+function handleError(error) {
```

Comment:
Is this function still needed if its just throwing if the parameter is truthy? Wouldn't it be better to throw the error directly?

Reply:
legacy code. Removed said function and modified other auth functions to remove try/catch. Catch of errors is handled by onSubmit in AuthForm.tsx where they call the auth functions.